build.sh

```diff
@@ -1016,38 +1022,35 @@ their entries in your Podfile.
 
 
 EOM
-            exit 1
-          fi
-
-          rm -rf include
-          mkdir -p include
-          mv core/include include/core
-
-          mkdir -p include/impl/apple include/util/apple include/sync/impl
-          cp Realm/*.hpp include
-          cp Realm/ObjectStore/src/*.hpp include
-          cp Realm/ObjectStore/src/sync/*.hpp include/sync
-          cp Realm/ObjectStore/src/sync/impl/*.hpp include/sync/impl
-          cp Realm/ObjectStore/src/impl/*.hpp include/impl
-          cp Realm/ObjectStore/src/impl/apple/*.hpp include/impl/apple
-          cp Realm/ObjectStore/src/util/*.hpp include/util
-          cp Realm/ObjectStore/src/util/apple/*.hpp include/util/apple
+          exit 1
+        fi
 
-          touch Realm/RLMPlatform.h
-          if [ -n "$COCOAPODS_VERSION" ]; then
-            # This variable is set for the prepare_command available
-            # from the 1.0 prereleases, which requires a different
-            # header layout within the header_mappings_dir.
-            cp Realm/*.h include
-          else
-            # For CocoaPods < 1.0, we need to scope the headers within
-            # the header_mappings_dir by another subdirectory to avoid
-            # Clang from complaining about non-modular headers.
-            mkdir -p include/Realm
-            cp Realm/*.h include/Realm
-          fi
+        rm -rf include
+        mkdir -p include
+        mv core/include include/core
+
+        mkdir -p include/impl/apple include/util/apple include/sync/impl
+        cp Realm/*.hpp include
+        cp Realm/ObjectStore/src/*.hpp include
+        cp Realm/ObjectStore/src/sync/*.hpp include/sync
+        cp Realm/ObjectStore/src/sync/impl/*.hpp include/sync/impl
+        cp Realm/ObjectStore/src/impl/*.hpp include/impl
+        cp Realm/ObjectStore/src/impl/apple/*.hpp include/impl/apple
+        cp Realm/ObjectStore/src/util/*.hpp include/util
+        cp Realm/ObjectStore/src/util/apple/*.hpp include/util/apple
+
+        touch Realm/RLMPlatform.h
+        if [ -n "$COCOAPODS_VERSION" ]; then
```

Comment:
It just occurred to me that changes to the podspec are likely to break backwards-compatibility to versions of CocoaPods before 1.0, as [documented in this commit](https://github.com/realm/realm-cocoa/commit/71bb89389d1784d1130f001a5986a0735cb896c8). That seems acceptable at this point, so that we could also drop this if branch.